### PR TITLE
send email

### PR DESCRIPTION
**Pull request by huuanh99**
## Related Tickets:
- [#44912](https://edu-redmine.sun-asterisk.vn/issues/44912)

## WHAT this PR does?
make send email order detail for user

## Notes Impacted Areas:
- N/A.

## Attachments:
im1
im2

### CODE REVIEW

**Comment by huuanh99** [comment]
ready

File: resources/views/emails/orders/reports.blade.php [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@component('mail::message')
+<h2>BẢN BÁO CÁO DOANH THU</h2>
+<h3>Doanh thu cửa hàng trong ngày {{ $day }}</h3>
+<h3>Tổng doanh thu : {{ vndFormat($total) }}</h3>
+Chúc bạn có 1 ngày làm việc hiệu quả,<br>
+{{ config('app.name') }}
+@endcomponent
```
Comment (ghost) [inline resources/views/emails/orders/reports.blade.php]: indent 4 space

**Review by ghost — COMMENTED** [review]
đính kèm ảnh Email cho m nha.

**Comment by huuanh99** [comment]
ready

**Pull request merged**